go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/kyma-project/infrastructure-manager
 
-go 1.21
+go 1.22.1
```

Comment:
Please change version to correct format 

`go 1.22`

Reply:
@koala7659  I have started with that version but it did not work. 

If I change it to 1.22
```
git diff go.mod
────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
modified: go.mod
────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
@ go.mod:3 @
module github.com/kyma-project/infrastructure-manager

- go 1.22.1
+ go 1.2
```

I get the error:

```
Error: load packages in root "/Users/i316752/SAPDevelop/infrastructure-manager": err: exit status 1: stderr: go: downloading go1.22 (darwin/arm64)
go: download go1.22 for darwin/arm64: toolchain not available
```

Found [here](https://github.com/golang/go/issues/65568#issuecomment-1932597413 ) that specifying minor version works since 1.22. It's AFAIU either that or adding `toolchain go1.22.1` to the `go.mod`.